upup/pkg/fi/cloudup/bootstrapchannelbuilder.go

```diff
@@ -532,7 +532,7 @@ func (b *BootstrapChannelBuilder) buildManifest() (*channelsapi.Addons, map[stri
 		versions := map[string]string{
 			"pre-k8s-1.6": "2.4.2-kops.1",
 			"k8s-1.6":     "2.4.2-kops.1",
-			"k8s-1.8":     "2.6.3-kops.2",
+			"k8s-1.8":     "2.6.8-kops.2",
```

Comment:
I believe hte version should be 2.6.7-kops.1 but would like to get @justinsb's confirmation.  If so, I suspect we should add the suffix to the calico addons above as well.

Reply:
Thanks for the confirmation. I've pushed an update fixing this.